libs/stack/stack-ui/src/components/TagGroup/tag-group.stories.jsx

```
@@ -0,0 +1,32 @@
+
+export const Default = Template.bind({})
+
+export const Disabled = {
+  render: Template.bind({}),
+  name: 'Disabled',
+  args: {
+    disabledKeys: ['red'],
+  },
+}
+
+export const Error = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)